rust/connlib/libs/common/Cargo.toml

```diff
@@ -31,8 +31,14 @@ rand = { version = "0.8", default-features = false, features = ["std"] }
 chrono = { workspace = true }
 parking_lot = "0.12"
 
+# Needed for Android logging until tracing is working
+log = "0.4"
```

Comment:
I think we should enable the [`log` feature](https://docs.rs/tracing/latest/tracing/#crate-feature-flags) from the tracing crate instead. then we can just drop `log` entirely and `tracing` will automatically emit `log`'s logs if we are on android.